### DISCUSSION TITLE
Show Mouse with pictures

### DISCUSSION THREAD

**Discussion by BK-Y** [Jagailo/YetAnotherKeyDisplayer#16] [Ideas]
If mouse-clicking operation could be shown with pictures, it will be awsome!

**Comment by Jagailo** [comment]
Hello Peiyang,

Thanks for using the app, but the output of pressed keys is displayed as text. This is a key feature of YAKD that won't change.
Unfortunately, it's not technically possible to display pictures with buttons.

Best regards.
